react/Payment.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useRef, useCallback } from 'react'
+import React, { useState, useRef, useCallback, useEffect} from 'react'
```

Comment:
weird that eslint didn't catch that 🤔
for me, it only catched when I ran prettier on this specific file. anyone knows why?

Anyway, if you feel like fixing it, ok, but not obligatory in my opinion